### DISCUSSION TITLE
作者你好，是否已经支持了grpc协议

### DISCUSSION THREAD

**Discussion by cfy3133** [arextest/arex-agent-java#535] [General]
作者你好，是否已经支持了grpc协议的录制回放，已经mock形式的测试功能

**Comment by mr3** [comment]
@cfy3133 Thank you for posting in the AREX community! We value your feedback and our team is paying attention to the feedback, feature requests & bugs we are hearing from users in the Community Discussions.

The GRPC protocol is not yet supported.

**Reply by cfy3133** [reply]
好的，感谢回复
